.github/ISSUE_TEMPLATE/2-Enhancement.md

```diff
@@ -0,0 +1,5 @@
+---
+name: Enhancement
```

Comment:
I'd call it `Suggestion` or `Feature Request`, but `Enhancement` works just as well I guess.

Reply:
My initial thought was that the term "Suggestion" discourages feature-request spam and also is more broadly applicable. But after giving it a second thought, feature request seems the way to go. Its commonly used in this context and could also be used to describe other improvements than new features. So I changed it to "Feature requests"